FileSystem/FileMapperMockerUtil.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use VBAFileMapperMocker file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Get File Helper function
+ * Gets the file id of a local file if available. Otherwise returns null.
+ * @param {string} localPath Local filepath of the file
+ * @return {string} File id of the local file if available, otherwise null
+ */
+function getFileIdUtil(localPath) {
+  // Get platform agnostic relative path split
+  var pathSplit = VBAFileMapperMocker.getRelativePathSplit(localPath);
+  // Get base directory
+  var root;
+  try {
+    root = getFinalFolderUtil(pathSplit);
+  } catch (e) {
+    return null;  // No folder found
+  }
+  // Obtain the file
+  var fileIterator = root.getFilesByName(pathSplit[pathSplit.length - 1]);
+  if (!fileIterator.hasNext()) {
+    return null;
+  }
+  return fileIterator.next().getId();
+}
+
+/**
+ * Get Folder Helper function
+ * Gets the file id of a local folder if available. Otherwise returns null.
+ * @param {string} localPath Local filepath of the folder
+ * @return {string} Folder id of the local folder if available, otherwise null
+ */
+function getFolderUtil(localPath) {
```

Comment:
Generally Util is used for class names, can we use getFolderId instead?

Reply:
Fixed